client/question/services/FeedbackGeneratorService.js

```diff
@@ -62,6 +93,40 @@ tie.factory('FeedbackGeneratorService', [
     var previousRuntimeErrorString = '';
 
     /**
+     * Pseudo-randomly returns an int between min and max (inclusive).
+     *
+     * @param {number} min Minimum number for range.
+     * @param {number} max Maximum number for range.
+     * @returns {number}
+     * @private
+     */
+    var _getRandomInt = function(min, max) {
+      min = Math.ceil(min);
+      max = Math.floor(max);
+      return Math.floor(Math.random() * (max - min)) + min;
+    };
+
+    /**
+     * Randomly selects feedback string from an object containing variations
+     * of a given feedback type.
+     *
+     * @param {Object.<string, Array.<string>>} CORRECTNESS_FEEDBACK_TEXT
+     * @param {string} type Correctness feedback type.
+     * @returns {string}
+     * @private
+     */
+    var _getCorrectnessFeedbackString = function(
+        CORRECTNESS_FEEDBACK_TEXT, type) {
```

Comment:
This wasn't what I meant when I wrote the previous comment, I think. What I was trying to say was that you do not need to pass in CORRECTNESS_FEEDBACK_TEXT as an argument at all.

Reply:
ah, got it

Reply:
Fixed